CMakeLists.txt

```diff
@@ -521,7 +521,7 @@ message(STATUS "QT_INSTALL_DIR=${QT_INSTALL_DIR}")
 
 find_package(Qt6 ${QT_VERSION} COMPONENTS CoreTools GuiTools WidgetsTools QmlTools WebEngineCoreTools REQUIRED PATHS ${QT_INSTALL_DIR} NO_DEFAULT_PATH)
 
-find_package(Qt6 ${QT_VERSION} COMPONENTS Core Core5Compat Gui Widgets Sql Network Xml Concurrent PrintSupport Quick QuickWidgets Qml WebChannel Positioning WebEngineCore WebEngineWidgets REQUIRED PATHS ${QT_INSTALL_DIR} NO_DEFAULT_PATH)
+find_package(Qt6 ${QT_VERSION} COMPONENTS Core Core5Compat Gui Widgets Sql Svg Network Xml Concurrent PrintSupport Quick QuickWidgets Qml WebChannel Positioning WebEngineCore WebEngineWidgets REQUIRED PATHS ${QT_INSTALL_DIR} NO_DEFAULT_PATH)
```

Comment:
We aren't using QSvg nor do we have any .svg files in the repo. Why would we need it?

Reply:
I am not 100% sure, Qt might be using SVG internally, I think some of our web assests use SVG.  The app does load that dll when it is installed so I figured it was good to just include it:

![image](https://user-images.githubusercontent.com/1327850/168589994-51532451-73f4-4168-8932-54eaeedd5deb.png)

Reply:
I just checked out the source code for windeployqt, what it does is to find some DLLs you do use, then it reads what this DLL depends on and includes them as well. (findDependentQtLibraries -> readExecutable -> readPeExecutable)